react/src/components/SettingList.tsx

```
@@ -24,11 +24,17 @@ const useStyles = createStyles(({ css }) => ({
 interface SettingPageProps {
   settingGroup: { title: string; settingItems: SettingItemProps[] }[];
   tabDirection?: 'top' | 'left';
+  isShowChangedOptionFilter?: boolean;
+  isShowResetButton?: boolean;
+  isShowSearchBar?: boolean;
```

For the prop name, it's better to use `show***` instead of `isShow***`.
<SomeComponent showResetButton />

Also, if the default is `show`, it's better to use `hide***` instead of `show***`.

---

Thank you! I'm going to check it.